libs/email/components/content.tsx

```
@@ -2,33 +2,125 @@
+                              {events.slice(1).map((event, idx) => (
+                                <tr key={idx}>
+                                  <td className="py-4">
+                                    <Card title={event.title}>
+                                      <Text className="text-sm text-[#e6e6e6] font-mono">
+                                        {event.text}
+                                      </Text>
+                                      {
+                                        event.buttons.map(button =>
+                                          (
+                                            <div key={button.text} style={{ marginBottom: 8 }}>
+                                              <a href={button.link} target="_blank" className="block no-underline text-black">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)